VideoApi/VideoApi.UnitTests/Controllers/Participant/UpdateConferenceParticipantsTests.cs

```diff
@@ -23,33 +23,22 @@ public void SetUp()
         {
             _existingParticipants = new List<UpdateParticipantRequest>
             {
-                new UpdateParticipantRequest
+                new()
```

Comment:
Might be worth deleting these unit tests.
If you ran the Integration Tests project with coverage, and these controllers are already covered then I would say that's enough evidence to drop unit tests already covered by the API tests

Reply:
there is no integration test coverage for that endpoint at present. I can create a tech debt ticket for it

I know we had conversations about replacing the unit tests with integration tests - maybe we can do it as part of that

Reply:
brilliant thanks!